src/pydocstyle.py

```diff
@@ -140,13 +144,17 @@ def is_empty_or_comment(line):
         return ''.join(reversed(list(filtered_src)))
 
     def __str__(self):
-        return 'in %s %s `%s`' % (self._publicity, self._human, self.name)
+        out = 'in %s %s `%s`' % (self._publicity, self._human, self.name)
```

Comment:
Should I just replace all instances, or might that create unnecessary merge
conflicts with other open PRs?